starfish/core/experiment/builder/__init__.py

```diff
@@ -28,13 +35,18 @@
 from .providers import FetchedTile, TileFetcher
 
 
-AUX_IMAGE_NAMES = {
-    'nuclei',
-    'dots',
-}
 DEFAULT_DIMENSION_ORDER = (Axes.ZPLANE, Axes.ROUND, Axes.CH)
 
 
+@dataclass
+class TileCoordinate:
+    """Data class for encapsulating the location of a tile in a"""
```

Comment:
```suggestion
    """Data class for encapsulating the location of a tile in an experiment"""
```

or at least I think that's what you meant.

Reply:
Fixed.